Gopkg.toml

```diff
@@ -9,7 +9,7 @@ required = [
 
 [[constraint]]
   name = "github.com/linkerd/linkerd2-proxy-api"
-  version = "v0.1.3"
+  revision = "de20a798ba70fcc27f91054088786b68fe0d9dcc" # FIXME
```

Comment:
We should not merge this until the linkerd2-proxy-api repo has a 0.1.4 release with this change.